fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -612,6 +775,30 @@ private static void generatePaths(Class<?> modelClass, JsonObjectBuilder paths,
         // TODO: add patch
```

Comment:
forgot about this TODO; wonder if we should open a placeholder issue for this one too...

Reply:
opened https://github.com/IBM/FHIR/issues/2202 for this...feel free to add.

I also realized we don't document our support for the "conditional" variants (conditional create, update, and delete), but I supposed we need to draw the line somewhere...